src/alda/Util.java

```diff
@@ -172,8 +174,8 @@ public static String getStdIn() {
   }
 
   public static String getProgramPath() throws URISyntaxException {
-    return Main.class.getProtectionDomain().getCodeSource().getLocation()
-               .toURI().getPath();
+    URI pathURI = Main.class.getProtectionDomain().getCodeSource().getLocation().toURI();
```

Comment:
I needed to change this because the windows filename being outputted was prefixed with a `/`, which the download script can't handle. This change should output it correctly for all platforms. See: https://stackoverflow.com/questions/6164448/convert-url-to-normal-windows-filename-java